common.lic

```diff
@@ -525,8 +525,15 @@ module DRC
   end
 
   def check_encumbrance(refresh = true)
-    bput('encumbrance', 'Encumbrance : .*') if refresh
-    $ENC_MAP[DRStats.encumbrance]
+    encumbrance = DRStats.encumbrance
+    if refresh
+      encumbrance_pattern = /(Encumbrance)\s:\s(?<encumbrance>.*)/
+      case bput('encumbrance', encumbrance_pattern)
+      when encumbrance_pattern
+        encumbrance = Regexp.last_match[:encumbrance]
+      end
+    end
+    $ENCUMBRANCE_MAP[encumbrance]
```

Comment:
Should this be `$ENC_MAP`?

Reply:
Yes... forgot I renamed that variable locally 

```suggestion
    $ENC_MAP[encumbrance]
```